src/generated/stellar-xdr_generated.ts

```diff
@@ -3881,4 +3881,6 @@ xdr.struct("HmacSha256Mac", [
 ]);
 
 });
-export default types;
+
+export default xdr;
+export { XdrNS as Xdr } from '../xdr_definitions';
```

Comment:
@Akuukis btw, I'm iffy about touching this file mostly because it is auto-generated so we don't have control over it